src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/wa/CftQueryServiceClaimTaskTest.java

```diff
@@ -227,6 +327,56 @@ void should_retrieve_a_task_when_grant_type_specific() {
 
     }
 
+    @Test
+    void should_retrieve_a_task_when_grant_type_specific_and_permission_own() {
```

Comment:
can we review test setup for this test with caseId "1623278362431004"? Important we stay aligned if we migrate FT to IT. E.g. an ftpa-judge cannot have own permission. for task (process application)